repos/uniPlanner-server/src/controllers/course/course.controller.ts

```diff
@@ -7,14 +7,39 @@ export class CourseController {
 
     constructor(readonly mongoService: MongoService) {}
 
-    @Get('course')
-    getCourseById(): Course {
+    @Post('SaveCourse')
+    saveItemByCollection(@Body() course): string {
```

Comment:
add async and chage to - async  saveItemByCollection(@Body() course): Promise<string>

change it in all functions that start without async